topics/ICRC-21/icrc_21_consent_msg.md

```diff
@@ -33,14 +33,10 @@ In addition to implementing this interface, it is recommended that the canister
 
 The signer may send the `icrc21_consent_message` call using the same identity as it would for the actual canister call for which the consent message was issued.
 
-Any canister implementing the `icrc21_consent_message` interface must not require authentication for this call.
-However, canisters may add additional or different information if a non-anonymous `sender` is used.
+Any canister implementing the `icrc21_consent_message` interface must not require authentication for this call. Anonymous consent messages are required in the [cold signer use-case](#cold-signer-use-case) which would otherwise require two interactions with the cold signer component, making the flow very cumbersome for users.
 
-For example, a canister might include private information in the consent message, if the call is made by the owner of that information. 
-
-The canister must ensure that if the actual call is made with a different identity than the `icrc21_consent_message` then either:
```

Comment:
I think this part was simply confusing. Removing it entirely and enforcing consistency is IMHO much cleaner.